scripts/fuzz_opt.py

```diff
@@ -358,12 +358,40 @@ def can_compare_to_others(self):
                 # C won't trap on OOB, and NaNs can differ from wasm VMs
                 return not OOB and not NANS
 
+        class Wasm2C2Wasm(Wasm2C):
+            name = 'wasm2c2wasm'
+
+            def run(self, wasm):
+                run([in_bin('wasm-opt'), wasm, '--emit-wasm2c-wrapper=main.c'] + FEATURE_OPTS)
+                run(['wasm2c', wasm, '-o', 'wasm.c'])
+                # pass-debug can be very slow in large emcc inputs
+                if os.environ.get('BINARYEN_PASS_DEBUG'):
+                    del os.environ['BINARYEN_PASS_DEBUG']
+                compile_cmd = ['emcc', 'main.c', 'wasm.c', os.path.join(self.wasm2c_dir, 'wasm-rt-impl.c'), '-I' + self.wasm2c_dir, '-lm']
```

Comment:
This is going to be a little annoying because I usually don't have an `emcc` on my path, but I guess I'll have to figure that out for myself. I usually don't have an `emcc` because the version I want varies so much.

Reply:
There might be a better option, but I'm not sure what it is. We could look at `~/.emscripten` but in general we want to avoid that global settings file.

Btw, it's fine to fuzz without emcc, that could be done rarely and only if its convenient for you.

Reply:
I don't see logic to skip the test if `emcc` is not found. Am I missing it?

Reply:
Good point, that was missing. Added now.